src/edu/csus/ecs/pc2/ui/EditFilterPane.java

```diff
@@ -319,10 +336,27 @@ private JPanel getTeamsPane() {
         }
         return teamsPane;
     }
-    
+
+    /**
+     * This method initializes teamFrame
```

Comment:
teamFrame or a groupPane?

Reply:
Yeah.  Fixed that one and a couple of other "legacy" cut&paste ones as well.
 [Commit](https://github.com/pc2ccs/pc2v9/pull/909/commits/ac4635f11971b2ebf29eb6f764a50d615603b0f4)